Scripts/2_CreateParentDisks.ps1

```diff
@@ -736,11 +782,11 @@ If (!( $isAdmin )) {
                     xDhcpServerOption MgmtScopeRouterOption
                     {
                         Ensure = 'Present'
-                        ScopeID = '10.0.0.0'
+                        ScopeID = ($DHCPscope+"0")
```

Comment:
Is this +"0" needed?

Reply:
Yes, in the beginning of the script I strip the last character:

$DHCPscope = $DHCPscope.Substring(0,$DHCPscope.Length-1) 

So on line 785 "$DHCPscope" equals 10.0.0.

Reply:
OK, so it means, that scope has to end with .0 (all /24 do :)

Reply:
That is correct. That is the reason why only a /24 is allowed.